src/responseHandlers/AbstractResponseHandler.js

```diff
@@ -33,6 +36,27 @@ var AbstractResponseHandler = jsface.Class([EventEmitter], {
 	},
 
 	_printResponse: function(error, response, body, request) {
+        if (Globals.outputFileVerbose) {
+            var filepath = path.resolve(Globals.outputFileVerbose);
+
+            var requestString = "-------------------------------------------------------------------------------------------\n" +
```

Comment:
Perhaps this could be extended to a broader discussion on the use of Mustache in Newman for templating output ?